### ISSUE TITLE
Handle totalTime keyword

### ISSUE THREAD

**Issue by Timvde** [mrzapp/nextcloud-cookbook#89]
I have encountered a website that uses only `totalTime` instead of splitting the time into `prepTime` and `cookTime`. This appears to be valid according to the spec. Currently, `totalTime` is completely ignored in the code.

Should it be an extra field? Do we just fill in the time in either `prepTime` or `cookTime`? What if all three are in the input?

**Comment by mrzapp** [comment]
@Timvde if totalTime is in the schema.org spec, then we should support it too. Are you interested in making a pull request yourself, or should we handle it? If you're interested, the sanity check code for the backend is [here](https://github.com/mrzapp/nextcloud-cookbook/blob/develop/lib/Service/RecipeService.php#L72), and the template for the editor is [here](https://github.com/mrzapp/nextcloud-cookbook/blob/develop/templates/content/edit.php#L30)

**Comment by Timvde** [comment]
Yes, it's part of the `HowTo`, which `Recipe` inherits from: https://schema.org/Recipe

I'm not a PHP dev, but I have some spare time in the following days, so I'll give it a shot.

My main question is: how can I easily test it out? Install the full Nextcloud stack including a webserver on my local machine and checkout this repository in the `apps` subdirectory?

**Comment by mrzapp** [comment]
@Timvde I just use my running NextCloud instance and make code changes via SSH and vim, so I'm always running bleeding edge.

The approach you mentioned is a lot more proper, although the directory is called `custom_apps`. If you don't want to install the full LAMP stack on your own machine, you can use Docker to keep things separated.

**Comment by christianlupus** [comment]
Just one more hint: Check the app out in a folder called `cookbook` not the default (by `git clone`) `nextcloud-cookbook`. Otherwise the install will not succeed (at least at my test installation).
